selfdrive/car/hyundai/values.py

```diff
@@ -181,8 +188,12 @@ def init_make(self, CP: car.CarParams):
     HyundaiCarInfo("Hyundai Palisade 2020-22", "All", video_link="https://youtu.be/TAnDqjF4fDY?t=456", car_parts=CarParts.common([CarHarness.hyundai_h])),
     HyundaiCarInfo("Kia Telluride 2020-22", "All", car_parts=CarParts.common([CarHarness.hyundai_h])),
   ],
+  CAR.PALISADE_2023: [
+    HyundaiCarInfo("Hyundai Palisade (without HDA II) 2023", "All", car_parts=CarParts.common([CarHarness.hyundai_a])),
+    HyundaiCarInfo("Kia Telluride (without HDA II) 2023", "All", car_parts=CarParts.common([CarHarness.hyundai_l])),
+  ],
   CAR.VELOSTER: HyundaiCarInfo("Hyundai Veloster 2019-20", min_enable_speed=5. * CV.MPH_TO_MS, car_parts=CarParts.common([CarHarness.hyundai_e])),
-  CAR.SONATA_HYBRID: HyundaiCarInfo("Hyundai Sonata Hybrid 2020-22", "All", car_parts=CarParts.common([CarHarness.hyundai_a])),
+  CAR.SONATA_HYBRID: HyundaiCarInfo("Hyundai Sonata Hybrid 2020-22", "All", car_parts=CarParts.common([CarHarness.hyundai_e])),
```

Comment:
why was this changed?

Reply:
I might have accidentally changed it when the generator ran. Will fix it during rebase.